include/bsp/xi_bsp_tls.h

```diff
@@ -180,6 +180,20 @@ xi_bsp_tls_state_t xi_bsp_tls_read( xi_bsp_tls_context_t* tls_context,
                                     uint8_t* data_ptr,
                                     size_t data_size,
                                     int* bytes_read );
+
+/**
+ * @function
+ * @brief Implements the TLS pending.
+ *
+ * Implementation of this function should call TLSs library pending function. It should
+ * return number of bytes which are buffered and available in the TLS library to be read
```

Comment:
It should return the the number of bytes that are available to be read by the tls library's read function.